### ISSUE TITLE
No trade after a loss (10 candles)

### ISSUE THREAD

**Issue by FagerCrypto** [freqtrade/freqtrade#9156]
Good evening, guys

I'd like to know how I can code that no trade shall be open after 10 candles if there's loss. In easy words. if loss, no trades for 10 candles. the next trade can be opened at the 11th candle. thanks for your help.

``` pragma pylint: disable=missing-docstring, invalid-name, pointless-string-statement

# flake8: noqa: F401
# isort: skip_file
# --- Do not remove these libs ---
import numpy as np  # noqa
import pandas as pd  # noqa
from pandas import DataFrame
from typing import Optional, Union
from datetime import datetime

from freqtrade.strategy import (BooleanParameter, CategoricalParameter, DecimalParameter,
                                IStrategy, IntParameter)

# --------------------------------
# Add your lib to import here
import talib.abstract as ta
import freqtrade.vendor.qtpylib.indicators as qtpylib


# This class is a sample. Feel free to customize it.
class MaturaarbeitStrategy(IStrategy):
    
    def leverage(self, pair: str, current_time: datetime, current_rate: float,
                 proposed_leverage: float, max_leverage: float, entry_tag: Optional[str], side: str,
                 **kwargs) -> float:
        
        
        return 5

    

    # Strategy interface version - allow new iterations of the strategy interface.
    # Check the documentation or the Sample strategy to get the latest version.
    INTERFACE_VERSION = 10

    

    # Minimal ROI designed for the strategy.
    # This attribute will be overridden if the config file contains "minimal_roi".
    minimal_roi = {
        
        "60": 0.2,
        "30": 0.3,
        "0": 0.4
    }

    # Optimal stoploss designed for the strategy.
    # This attribute will be overridden if the config file contains "stoploss".
    stoploss = -0.8

    # Trailing stoploss
    trailing_stop = False
    # trailing_only_offset_is_reached = False
    # trailing_stop_positive = 0.01
    # trailing_stop_positive_offset = 0.0  # Disabled / not configured

    # Optimal timeframe for the strategy.
    timeframe = '5m'

    # Run "populate_indicators()" only for new candle.
    process_only_new_candles = True

    # These values can be overridden in the config.
    use_exit_signal = False
    exit_profit_only = True
    ignore_roi_if_entry_signal = False

    # Hyperoptable parameters
    buy_rsi = IntParameter(low=1, high=50, default=30, space='buy', optimize=True, load=True)
    sell_rsi = IntParameter(low=50, high=100, default=70, space='sell', optimize=True, load=True)
    short_rsi = IntParameter(low=51, high=100, default=70, space='sell', optimize=True, load=True)
    exit_short_rsi = IntParameter(low=1, high=50, default=30, space='buy', optimize=True, load=True)

    # Number of candles the strategy requires before producing valid signals
    startup_candle_count: int = 30

    # Optional order type mapping.
    order_types = {
        'entry': 'limit',
        'exit': 'limit',
        'stoploss': 'market',
        'stoploss_on_exchange': False
    }

    # Optional order time in force.
    order_time_in_force = {
        'entry': 'GTC',
        'exit': 'GTC'
    }

    plot_config = {
        'main_plot': {
            'ema5': {'color': 'violet'}
            
        },
        'subplots': {
            "MACD": {
                'macd': {'color': 'blue'},
                'macdsignal': {'color': 'orange'},
            },
            "RSI": {
                'rsi': {'color': 'red'},
                
            },
            "ADX": { 
                'adx': {'color': 'black'},
                'plus_di' : {'color': 'green'},
                'plus_dm' : {'color': 'green'},
                'minus_di' : {'color': 'red'},
                'minus_dm' : {'color': 'red'},
            },
            
                
            }
        
        
        }
    



    def populate_indicators(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
        """
        Adds several different TA indicators to the given DataFrame

        Performance Note: For the best performance be frugal on the number of indicators
        you are using. Let uncomment only the indicator you are using in your strategies
        or your hyperopt configuration, otherwise you will waste your memory and CPU usage.
        :param dataframe: Dataframe with data from the exchange
        :param metadata: Additional information, like the currently traded pair
        :return: a Dataframe with all mandatory indicators for the strategies
        """

        # Momentum Indicators
        # ------------------------------------

        # ADX
        dataframe['adx'] = ta.ADX(dataframe)

        # # Plus Directional Indicator / Movement
        dataframe['plus_dm'] = ta.PLUS_DM(dataframe)
        dataframe['plus_di'] = ta.PLUS_DI(dataframe)

        # # Minus Directional Indicator / Movement
        dataframe['minus_dm'] = ta.MINUS_DM(dataframe)
        dataframe['minus_di'] = ta.MINUS_DI(dataframe)

       
        # RSI
        dataframe['rsi'] = ta.RSI(dataframe)

      

     
        # MACD
        macd = ta.MACD(dataframe)
        dataframe['macd'] = macd['macd']
        dataframe['macdsignal'] = macd['macdsignal']
        dataframe['macdhist'] = macd['macdhist']

        # MFI
        dataframe['mfi'] = ta.MFI(dataframe)

      


        # Bollinger Bands
        bollinger = qtpylib.bollinger_bands(qtpylib.typical_price(dataframe), window=20, stds=2)
        dataframe['bb_lowerband'] = bollinger['lower']
        dataframe['bb_middleband'] = bollinger['mid']
        dataframe['bb_upperband'] = bollinger['upper']
        dataframe["bb_percent"] = (
            (dataframe["close"] - dataframe["bb_lowerband"]) /
            (dataframe["bb_upperband"] - dataframe["bb_lowerband"])
        )
        dataframe["bb_width"] = (
            (dataframe["bb_upperband"] - dataframe["bb_lowerband"]) / dataframe["bb_middleband"]
        )


        # # EMA - Exponential Moving Average
 
        dataframe['ema5'] = ta.EMA(dataframe, timeperiod=5)
    

     
     
        

        return dataframe

    def populate_entry_trend(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
        """
        Based on TA indicators, populates the entry signal for the given dataframe
        :param dataframe: DataFrame
        :param metadata: Additional information, like the currently traded pair
        :return: DataFrame with entry columns populated
        """
        
        dataframe.loc[   #qtpylib.crossed_above(
            (
                # Signal: RSI crosses above 30
               (dataframe['rsi'] < self.buy_rsi.value) &
               (dataframe['bb_lowerband'] > dataframe["low"]) &
               (dataframe['ema5'] > dataframe["close"]) &  # Guard: tema below BB middle
                (dataframe['macd'] < dataframe['macdsignal']) &
                #(dataframe['plus_dm'] < dataframe['minus_dm']) &
                (dataframe['plus_di'] < dataframe['minus_di']) &
                (dataframe['adx'] > 20)
                   # Make sure Volume is not 0
            ),
            'enter_long'] = 1

        dataframe.loc[
            (
                # Signal: RSI crosses above 70
                (dataframe['rsi'] > self.sell_rsi.value) &
                (dataframe['bb_upperband'] < dataframe["high"]) &
                (dataframe['ema5'] < dataframe["close"]) &  # Guard: tema below BB middle
                (dataframe['macd'] > dataframe['macdsignal']) & 
                #(dataframe['plus_dm'] > dataframe['minus_dm']) &
                (dataframe['plus_di'] > dataframe['minus_di']) &
                (dataframe['adx'] > 20)  # Make sure Volume is not 0
            ),
            'enter_short'] = 1

        return dataframe

    def populate_exit_trend(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
        """
        Based on TA indicators, populates the exit signal for the given dataframe
        :param dataframe: DataFrame
        :param metadata: Additional information, like the currently traded pair
        :return: DataFrame with exit columns populated
        """
    

        return dataframe
```

**Comment by xmatthias** [comment]
Please get used to filling out the issue template.  Those who frequently omit the template may have their issues closed without comment.
We're asking some very basic questions - by deleting the issue template, you're explicitly choosing not to answer our questions.

Filling out the template helps reduce our workload, and we greatly appreciate that.

---

Please read the [documentation](https://www.freqtrade.io/).
Based on this (and your prior) questions, please go through the full documentation at least once to know what freqtrade offers and can do.

While we're happy to help - we'll not be writing your matriculation project for you.

You'll probably want to look at [protections](https://www.freqtrade.io/en/latest/plugins/#protections). There's however also other methods to accomplish this which will provide more flexibility (you'll find that in the docs ...) :)
